app/helpers/ui_constants.rb

```diff
@@ -492,6 +492,10 @@ module UiConstants
                       {:tables => ui_lookup(:tables => "miq_server")}
                     end,
   }
+  ASSIGN_TOS["MiddlewareServer"] = {
```

Comment:
No.
It is a hash with class names by keys.
In this case, we are just following the same patter of the class.